app/policies/wrangling_policy.rb

```diff
@@ -2,14 +2,14 @@
 
 class WranglingPolicy < ApplicationPolicy
   FULL_ACCESS_ROLES = %w[superadmin tag_wrangling].freeze
-  VIEW_ACCESS_ROLES = (FULL_ACCESS_ROLES + %w[policy_and_abuse]).freeze
+  READ_ACCESS_ROLES  = (FULL_ACCESS_ROLES + %w[policy_and_abuse]).freeze
```

Comment:
Please apply the Rubocop suggestion, just a quick space removal:
```suggestion
  READ_ACCESS_ROLES = (FULL_ACCESS_ROLES + %w[policy_and_abuse]).freeze
```
(for files that don't have other violations you can also do `rubocop -a <path/to/file>`, but that gets tricky with some old files that need loads of fixes)